docs/en/tutorials/config.md

```diff
@@ -245,180 +166,283 @@ model = dict(
-        to_rgb=True),
+        type='PackDetInputs',  # Pipeline that formats the annotation data and decides which keys in the data should be packed into data_samples
+        meta_keys=('img_id', 'img_path', 'ori_shape', 'img_shape',
+                   'scale_factor'))
+]
+train_dataloader = dict(   # Train dataloader config
+    batch_size=2,  # Batch size of a single GPU
+    num_workers=2,  # Worker to pre-fetch data for each single GPU
+    persistent_workers=True,  # If ``True``, the dataloader will not shutdown the worker processes after an epoch end, which can accelerate training speed.
+    sampler=dict(  # training data sampler
+        type='DefaultSampler',  # DefaultSampler which supports both distributed and non-distributed training. Refer to https://github.com/open-mmlab/mmengine/blob/main/mmengine/dataset/sampler.py
+        shuffle=True),  # randomly shuffle the training data in each epoch
+    batch_sampler=dict(type='AspectRatioBatchSampler'),  # batch sampler for grouping images with similar aspect ratio into a same batch. It can reduce GPU memory cost.
+    dataset=dict(  # Train dataset config
+        type=dataset_type,
+        data_root=data_root,
+        ann_file='annotations/instances_train2017.json',  # Path of annotation file
+        data_prefix=dict(img='train2017/'),  # Prefix of image path
+        filter_cfg=dict(filter_empty_gt=True, min_size=32),  # config of filtering images and annotations
+        pipeline=train_pipeline))
+val_dataloader = dict(  # Validation dataloader config
+    batch_size=1,  # Batch size of a single GPU. If batch-szie > 1, the extra padding area may influence the performance.
+    num_workers=2,  # Worker to pre-fetch data for each single GPU
+    persistent_workers=True,  # If ``True``, the dataloader will not shutdown the worker processes after an epoch end, which can accelerate training speed.
+    drop_last=False,  # Whether to drop the last incomplete batch, if the dataset size is not divisible by the batch size
+    sampler=dict(
+        type='DefaultSampler',
+        shuffle=False),  # not shuffle the during validation and testing
+    dataset=dict(
+        type=dataset_type,
+        data_root=data_root,
+        ann_file='annotations/instances_val2017.json',
+        data_prefix=dict(img='val2017/'),
+        test_mode=True,  # Turn on test mode of the dataset to avoid filtering annotations or images
+        pipeline=test_pipeline))
+test_dataloader = val_dataloader  # Testing dataloader config
+```
+
+[Evaluators](https://mmengine.readthedocs.io/en/latest/tutorials/metric_and_evaluator.html) are used to compute the metrics of the trained model on the validation and testing datasets. The config of evaluators consists of one or a list of metric configs:
+
+```python
+val_evaluator = dict(  # Validation evaluator config
+    type='CocoMetric',  # The coco metric used to evaluate AR, AP, and mAP for detection and instance segmentation
```

Comment:
add the code link to `CocoMetric`

Reply:
CocoMetric will be moved to mmeval later.